conf/logback.xml

```diff
@@ -14,13 +14,6 @@
         </encoder>
     </appender>
 
-    <appender name="ACCESS_LOG_FILE" class="ch.qos.logback.core.FileAppender">
-        <file>logs/access.log</file>
```

Comment:
we are referring this ACCESS_LOG_FILE in the same file, we can remove that entry too.

Reply:
Removed the other occurrence @rambabu-posa  :-)